.github/CODEOWNERS

```diff
@@ -276,6 +276,7 @@
 /core/templates/dev/head/components/skill-mastery/ @sophiewu6 @vinitamurthi
 /core/templates/dev/head/components/skills-mastery-list/ @sophiewu6 @vinitamurthi
 /core/templates/dev/head/domain/question/ @aks681 @vinitamurthi
+/core/templates/dev/head/domain/review_test/ @nishantwrp
```

Comment:
Should @vinitamurthi @sophiewu6 also be added to this list?

Reply:
Ok I will add them